ctapipe/image/image_processor.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         self,
         subarray: SubarrayDescription,
         is_simulation,
+        use_telescope_frame=True,
```

Comment:
This should be a Traitlet, not a construction option.   Otherwise it is not configurable. The default should be set to True.

Reply:
That will not change the data model - it is only a configuration option, and the current data model supports both units in the output. However, we may in the future deprecate the CameraFrame one and clean up all that code as I said before, but now we want to be able to run the full analysis twice on a large dataset and ensure they results are compatible with both frames.

Reply:
While we are at it, please also remove the `is_simulation` option. It should not be needed. Code that needs to check that should just check `if event.simulation is None`

Reply:
Sorry, not really connected. Will just make a PR